Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fakebook.Profile.Domain
+{
+    public class DomainProfile
+    {
+        //[anything]@[anything].[anything]
+        public string Email { get; set; }
+        //should be a url
+        //defualts to a default image.
+        public Uri ProfilePictureUrl { get; set; }
+        
+
+        public string Name
+        {
+            get
+            {
+                return FirstName+' '+LastName;
+            }
+        }
+
+        //A-Z, ', ., - only... probs
```

Comment:
no value is gained by restricting characters from names, only headaches when you find out later you need them